src/components/UI/AttachmentLink/AttachmentLink.tsx

```diff
@@ -57,18 +57,38 @@ export const getExtension = (mimeType: string | undefined): React.ReactNode => {
   }
 };
 
+//sending message to child window
+const openTab = (data: string) => {
+  const childWin = window.open(`${window.location}/#verify-documents`, "_blank");
+  window.addEventListener(
+    "message",
+    (event) => {
+      if (event.data.type == "READY" && childWin) {
+        childWin.postMessage(
+          {
+            type: "LOAD_DOCUMENT",
+            payload: data,
+          },
+          `${window.location.href}`
+        );
+      }
+    },
+    false
+  );
+};
+
 export const AttachmentLink = ({ filename, data, type, path }: AttachmentLinkProps) => {
   let filesize = "0";
-  let redirectLink = "";
+  let isTT = filename.endsWith(".tt");
```

Comment:
const canOpenFile = isOpenAttestationFile(data);

Can write a function to test if it's an OA file (regardless if it ends with .tt), and use it below to render. This will make the code easier to read.